omnisafe/utils/command_app.py

```diff
@@ -99,11 +99,11 @@ def train(  # pylint: disable=too-many-arguments
     }
     keys = custom_cfgs[0::2]
     values = list(custom_cfgs[1::2])
-    custom_cfgs = dict(zip(keys, values))
-    custom_cfgs.update({'logger_cfgs:log_dir': os.path.join(log_dir, 'train')})
+    custom_cfgs_dict = dict(zip(keys, values))
```

Comment:
Useless `list` conversion. Also, Do we need to check that `len(custom_cfgs) % 2 == 0`?